example_test.py

```diff
@@ -72,12 +73,6 @@ def test_create_file(self):
 
     def test_delete_file(self):
         """Test example.delete_file()
-
```

Comment:
Instead of eliminating these comments entirely, perhaps add something to this effect to the module docstring at the top of the file:

Fake filesystem functions like `CreateFile()`, `CreateDirectory()` and `CreateLink()` are commonly used to set up file structures used in tests.  Of course you can also use the familiar `open()`, `os.mkdirs()` and so on.  `CreateFile()` is particularly convenient because it creates all the directories in the path, and allows you to specify the file contents or size.